src/bin/chess_cmd_vs_stockfish.rs

```diff
@@ -13,9 +13,9 @@ fn color_to_str(c: Option<GameColor>) -> String {
 }
 
 fn main() {
-    let mut player1 = ChessPlayerCmd::new();
-    let value_func = Box::new(StockfishNet::new());
-    let mut player2 = MCTSPlayer::new_custom(100000, 1.41421, value_func);
+    let mut player1 = ChessPlayerCmd {};
```

Comment:
Clippy recommendation is against "new()" with no argument, as the default constructor "MyClass {}" is still visible to the user, and it might skip some critical code in "new()".
The recommendation is to implement some trait that controls the behavior of "MyClass {}".
In such cases in our code, the "new()" didnt actually did anything, so it was easier to just remove it and use the default constructor

Reply:
Ahh okay, it takes a bit of getting used to (my first thought was - okay, then make the `MyClass {}` invisible from outside! But that's not how it works, it seems like initiating a class "directly" is legit "by definition")
BTW was is the trait you mentioned?

Reply:
the 'Default' trait